config/kafka-instance-types-configuration.yaml

```diff
@@ -12,6 +12,23 @@
 #   - [required] ams_resource: AMS resource to be used. Accepted values: ['rhosak']
 #   - [required] ams_product: AMS product to be used. Accepted values: ['RHOSAK', 'RHOSAKTrial', 'RHOSAKEval']
 #   - [required] ams_billing_models: List of AMS available billing models: Accepted values: ['marketplace', 'marketplace-rhm', 'marketplace-aws']
+#   - [optional] grace_period_days: The length of time in days when a Kafka instance is set to be in a 'grace period'. This configuration only applies
```

Comment:
Let's document the default too and the meaning of it

Reply:
I see it's documented a little bit below with another wording

Reply:
I've also added the default and minimum value at the top to make those clearer.